From 070fc5916b28e1197840f45b9142d57420d00738 Mon Sep 17 00:00:00 2001
From: Ivan Ocampo <ivanocampo07@gmail.com>
Date: Wed, 12 Nov 2025 16:56:04 -0500
Subject: [PATCH] firstFile.txt was editing

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index c667578..36ec56c 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-one line
-two line
+one 
+two 
 three line
 four line
 five line
-- 
2.43.0

